httpchecker.go

```diff
@@ -54,6 +54,23 @@ type HTTPChecker struct {
 	// requests. If not set, DefaultHTTPClient is
 	// used.
 	Client *http.Client `json:"-"`
+
+	// Headers is a slice of Headerthat contains the
+	// name and value for the header.
+	// Default is no additional headers are sent.
+	Headers []Header `json:"headers,omitempty"`
```

Comment:
Because of the line breaks?

```go
Headers: http.Header{
    "Content-Type": []string{"application/json"},
 },
```

I think we should stick to the std lib as much as possible (and I think this is reasonable).